stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -234,14 +243,65 @@ public DefaultDumperPluginsBuilder remove(String pluginName) {
     }
   }
 
+  /**
+   * Configuration mechanism to customize the behaviour of the standard set of inspector
+   * modules satisfying the Chrome DevTools protocol.  Note that while it is still technically
+   * possible to manually control these modules, this API is strongly discouraged and will not
+   * necessarily be supported in future releases.
+   */
   public static final class DefaultInspectorModulesBuilder {
-    private final Context mContext;
+    private final Application mContext;
```

Comment:
I sometimes do it this way to imply that most users don't actually care that it is an application specifically but at least one user does.  It makes it easier to refactor later when that user is refined or isolated away.